tests/run-e2e.sh

```diff
@@ -89,19 +92,36 @@ gather_olm() {
 	done
 }
 
-run_bundle() {
+run_bundle_upgrade() {
 	header "Running Bundle"
 	local -i ret=0
 
-	run ./tmp/bin/operator-sdk run bundle "$BUNDLE_IMG" \
-		--install-mode AllNamespaces --namespace "$OPERATORS_NS" --skip-tls || {
-		ret=$?
+	local replaced_version=""
+	replaced_version=$(yq ".spec.replaces| sub(\"$OPERATOR.v\"; \"\")" "$OPERATOR_CSV")
+
+	local released_bundle="$OPERATOR_RELEASED_BUNDLE:$replaced_version"
 
-		fail "Running Bundle failed"
+	info "Running Released Bundle - $replaced_version"
+	run ./tmp/bin/operator-sdk run bundle "$released_bundle" \
+		--install-mode AllNamespaces --namespace "$OPERATORS_NS" || {
+		ret=$?
+		line 50 heavy
 		gather_olm || true
+		fail "Running Released Bundle failed"
+		return $ret
 	}
 
-	return $ret
+	info "Running Upgrade to new bundle"
+	run ./tmp/bin/operator-sdk run bundle-upgrade "$BUNDLE_IMG" \
+		--namespace "$OPERATORS_NS" --use-http || {
```

Comment:
why use `--use-http` ?

Reply:
because localhost:5000 is using http

Reply:
oh got it

Reply:
https://github.com/operator-framework/operator-sdk/blob/master/internal/olm/operator/registry/index_image.go#L131